appveyor.ini

```diff
@@ -13,30 +12,40 @@ CreateCache = False
 # Settings applicable for all Crafts matrices
 # Settings are Category/key=value
 # Category is case sensitive
-
 [GeneralSettings]
-General/EMERGE_PKGDSTDIR=${Variables:APPVEYOR_BUILD_FOLDER}/binaries
-Paths/python = C:\Python36
-Paths/python27 = C:\Python27
+
+## This is the location of your python installation.
+## This value must be set.
+Paths/Python = C:\Python39-x64
+Paths/Python27 = C:\Python27-x64
```

Comment:
@mgallien I am confused by this path. Why adding `-x64`? will it get parsed by AppVeyor? I have my Python27 installed in `C:\Python27` without `-x64` postfix.

Reply:
this is the default path inside the appveyor image we use to do the build (documented here [Appveyor documentation](https://www.appveyor.com/docs/windows-images-software/#python) )